README.md

```diff
@@ -20,7 +20,7 @@ This is Nougaro beta 0.15.
 
  The code is compiled for Windows and Linux. Check it out in the [releases](https://github.com/jd-develop/nougaro/releases/) tab!
  
- Minimum : Python 3.10
+ Minimum : Python 3.11
  
  Supports Python 3.11
```

Comment:
Maybe change this line to "support Python 3.12" and delete the "Didn’t test under python 3.12" line. Don’t forget to change it in french too